llvm/lib/Target/RISCV/RISCVInstrInfoV.td

```diff
@@ -1814,5 +1814,61 @@ let Predicates = [HasVInstructionsI64, IsRV64] in {
   }
 } // Predicates = [HasVInstructionsI64, IsRV64]
 
+
+// The experimental zvzip extension.  Note these opcodes are very likely to
+// change!
+let Predicates = [HasStdExtZvzip], hasSideEffects = 0, mayLoad = 0,
+  mayStore = 0, isCodeGenOnly = 0, Inst<6-0> = OPC_CUSTOM_2.Value in {
```

Comment:
Do you know if anyone has looked for space in a standard opcode?

Reply:
Not yet.  It looks like I'm going to be driving this proposal, so that's one of my tasks for the next couple weeks.  This is why there's a prominent warning on this WIP about these being likely to change.